src/main/java/hockeyapp/HockeyappRecorder.java

```diff
@@ -82,7 +78,6 @@
     @DataBoundConstructor
     public HockeyappRecorder(List<HockeyappApplication> applications, boolean debugMode,
                              BaseUrlHolder baseUrlHolder, boolean failGracefully) {
-        this.schemaVersion = SCHEMA_VERSION_NUMBER;
```

Comment:
Seems odd that removing the version would fix, but more likely to believe team of people at Jenkins World :)